I meant that you should run flake8 for the formatting stuff:

```
$ flake8 plugins/base_commands.py 
plugins/base_commands.py:41:80: E501 line too long (156 > 79 characters)
plugins/base_commands.py:43:80: E501 line too long (126 > 79 characters)
plugins/base_commands.py:44:5: E301 expected 1 blank line, found 0
plugins/base_commands.py:48:80: E501 line too long (100 > 79 characters)
plugins/base_commands.py:50:38: E261 at least two spaces before inline comment
plugins/base_commands.py:50:39: E262 inline comment should start with '# '
plugins/base_commands.py:52:80: E501 line too long (124 > 79 characters)
plugins/base_commands.py:53:5: E301 expected 1 blank line, found 0
```

--- a/plugins/base_commands.py
+++ b/plugins/base_commands.py
@@ -39,12 +39,12 @@ class BaseCommandsPlugin(PluginBase):
             args = data['args']
             self.tpa_reqs[args[0]] = data['name']
             self.net.push_packet('PLAY>Chat Message',
-                                 {'message': '/tell ' + ''.join(args[0]) +
+                                 {'message': '/tell ' + args[0] +
                                   ' would like to tpa to you, type '
                                   + '(!)tpaccept or (!)tpdeny'})
         except IndexError:
             self.net.push_packet('PLAY>Chat Message',
-                                 {'message': '/tell ' + ''.join(data['name']) +
+                                 {'message': '/tell ' + data['name'] +
                                   ' Usage: (!)tpa [name]'})
 
     def handle_tpaccept(self, event, data):

--- a/plugins/base_commands.py
+++ b/plugins/base_commands.py
@@ -38,18 +38,29 @@ class BaseCommandsPlugin(PluginBase):
         try:
             args = data['args']
             self.tpa_reqs[args[0]] = data['name']
-            self.net.push_packet('PLAY>Chat Message', {'message': '/tell ' + ''.join(args[0]) + ' would like to tpa to you, type (!)tpaccept or (!)tpdeny'})
+            self.net.push_packet('PLAY>Chat Message',
+                                 {'message': '/tell ' + ''.join(args[0]) +
+                                  ' would like to tpa to you, type '
+                                  + '(!)tpaccept or (!)tpdeny'})
         except IndexError:
-            self.net.push_packet('PLAY>Chat Message', {'message': '/tell ' + ''.join(data['name']) + ' Usage: (!)tpa [name]'})
+            self.net.push_packet('PLAY>Chat Message',
+                                 {'message': '/tell ' + ''.join(data['name']) +
+                                  ' Usage: (!)tpa [name]'})
+
     def handle_tpaccept(self, event, data):
         to_who = data['name']
         try:
             from_who = self.tpa_reqs[to_who]
-            self.net.push_packet('PLAY>Chat Message', {'message': '/tp ' + from_who + ' ' + to_who})
+            self.net.push_packet('PLAY>Chat Message',
+                                 {'message': '/tp ' + from_who +
+                                  ' ' + to_who})
             logger.debug(json.dumps(self.tpa_reqs))
-            del self.tpa_reqs[to_who] #clear the table.
+            del self.tpa_reqs[to_who]
         except:
-            self.net.push_packet('PLAY>Chat Message', {'message': '/tell ' + to_who + ' you have no pending tpa requests.'})
+            self.net.push_packet('PLAY>Chat Message',
+                                 {'message': '/tell ' + to_who +
+                                  ' you have no pending tpa requests.'})
+
     def handle_jump(self, event, data):
         self.physics.jump()
 

